extension/src/plots/model/collect.ts

```diff
@@ -139,7 +141,9 @@ const collectImageData = (
   path: string,
   plot: ImagePlot
 ) => {
-  const pathLabel = path
+  const pathLabel = MULTI_IMAGE_PATH_REG.test(path)
+    ? (getParent(getPathArray(path), 0) as string)
```

Comment:
[Q] Can we make this another function? `getMultiImagePath` or something like that?